docs/examples/testing.rst

```diff
@@ -397,16 +397,20 @@ and :meth:`.setUp()`.
 	@classmethod
 	def setUpClass(cls):
 	    cls.cw = dtt.connectionwrapper()
-	    cls.initial = dtt.Table("book", """
+   	    cls.initial = dtt.Table("book", """
 	    | bid:int (pk) | title:text            | genre:text |
 	    | ------------ | --------------------- | ---------- |
 	    | 1            | Unknown               | Unknown    |
 	    | 2            | Nineteen Eighty-Four  | Novel      |
 	    | 3            | Calvin and Hobbes One | Comic      |
 	    | 4            | The Silver Spoon      | Cookbook   |""")
 
```

Comment:
Methods inside a class should be surrounded by [singular blank lines](https://peps.python.org/pep-0008/#blank-lines) according to PEP 8.

Reply:
Fixed